src/index.html

```diff
@@ -12,11 +13,22 @@
 		"description": "GodSVG is a free and open-source vector graphics application for structured SVG editing, available on all major desktop platforms and on web.",
 		"operatingSystem": ["Windows", "Linux", "MacOS", "Android"],
 		"applicationCategory": "GraphicsApplication",
-		"downloadUrl": "https://github.com/MewPurPur/GodSVG/releases/",
+		"url": "{{ site.url }}",
+		"downloadUrl": "{{ site.url }}/releases",
+		"image": "{{ site.url }}/assets/card.webp",
 		"softwareVersion": "{{ godsvg.version }}",
 		"author": {
 			"@type": "Person",
 			"name": "MewPurPur"
+		},
+		"offers": {
+			"@type": "Offer",
+			"price": 0
+		},
+		"aggregateRating": {
+			"@type": "AggregateRating",
+			"ratingValue": 4.0,
+			"ratingCount": 2
```

Comment:
![image](https://github.com/user-attachments/assets/f49df276-18aa-42b1-9c7c-c2a9de71af25)

Let's take this for granted.

Reply:
Also add something like "// Default 5-star scale
"bestRating": 5,
    "worstRating": 1,"